singer-io/tap-asana
# v.2.0.4 New endpoints, fields

# Description of change
Added endpoints: `portfolios`, `sections`, `stories`, and `teams`. Add missing fields to other endpoints.
Includes changes from v.2.0.3 PR (https://github.com/singer-io/tap-asana/pull/26).

# Manual QA steps
Ran singer-discover, singer-check-tap, target-stitch (both initial load and incremental sync) for ALL endpoints. No errors.
 
# Risks
Adds several endpoints and a few additional missing fields to existing endpoints. Maybe this should be a major version release?
 
# Rollback steps
 - revert this branch
